### ISSUE TITLE
epic: Transparent access to physical cluster pod definitions and sub-resources 

### ISSUE THREAD

**Issue by davidfestal** [kcp-dev/kcp#1916]
### TL;DR

Provide access to Physical cluster Pods definitions and sub-resources transparently through KCP at the expected API endpoint  

### Progress tracking <a href="#progress-tracking" id="progress-tracking">#</a>

EPIC detailed issues and overall progress can be tracked with the following [project view](https://github.com/orgs/kcp-dev/projects/1/views/39)

### Demo Objective

=== first step

- [x] user can use the various POD sub-resources (logs, exec, etc ...) through the expected resource Path of the user KCP workspace: `kubectl get --raw /clusters/root:org:ws/api/v1/namespaces/default/pods/foo/logs`
- [x] This path should be subject to the standard KCP Authrorization, so that a user should have expected RBAC permissions on the POD sub-resources to use it.

=== second step

- [x] user has access to the proxied `pods` (to list, get, etc ...) directly from the expected URL, so that he can do: `kubectl get pods`, and also `kubectl logs <pod>`

### Action Items

- [X] Scope of the current demo is necessary to fit in the prototype boundaries
- [ ] Contribute to the final demo script and recording

### Work Items

- [x] #1976
- [x] #1977
